assets/js/modules/tagmanager/setup.js

```diff
@@ -211,31 +211,28 @@ class TagmanagerSetup extends Component {
 				containerId: selectedContainer,
 			};
 
-			return await data.set( 'modules', 'tagmanager', 'save', optionData )
-				.then( ( responseData ) => {
-
-					if ( finishSetup ) {
-						finishSetup();
-					}
-
-					googlesitekit.modules.tagmanager.settings = {
-						accountId: responseData.accountId,
-						containerId: responseData.containerId,
-					};
-					if ( this._isMounted ) {
-						this.setState( {
-							isSaving: false
-						} );
-					}
-				} );
-		} catch ( err ) {
+			const responseData = await data.set( 'modules', 'tagmanager', 'save', optionData );
+			if ( finishSetup ) {
+				finishSetup();
+			}
+
+			googlesitekit.modules.tagmanager.settings = {
+				accountId: responseData.accountId,
+				containerId: responseData.containerId,
+			};
+
 			if ( this._isMounted ) {
 				this.setState( {
-					isLoading: false,
```

Comment:
Double checking, is this properly unset still? Seems critical to verify, as the component shouldn't remain in loading state when an error occurs.

Reply:
Yes, loading is true while requesting accounts and containers and set to false once they are loaded. 
Saving doesn't set `isLoading` to `true`.